packages/oauth/package.json

```diff
@@ -24,6 +24,7 @@
     "clean": "rm -rf lib dist types build tsconfig.tsbuildinfo",
     "fix-lint": "eslint . --fix && dprint fmt --config $(find-up dprint.json)",
     "lint": "eslint . && dprint check  --config $(find-up dprint.json)",
+    "test": "vitest run --pool=forks",
```

Comment:
I think Tim's changes were not properly failing build due to this not being present.